src/modal/index.html

```diff
@@ -0,0 +1,8 @@
+<div class="{ modal__invisible: $modal.visible }">
```

Comment:
I think we want this class to be `modal--invisible` to indicate a modifier instead of an element